### PR TITLE
Microsoft.DataBoxEdge RP: Corrected comments of trigger and extendedinfo APIs, corrected examples

### PR DESCRIPTION
**Pull request by anponnet**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by praries880** [comment]
@anponnet You are removing an operation from a stable API... this is a breaking change. We will have to wait for ARM's feedback on this change.

Why not bump up the API version here instead of modifying an existing Stable API version?

**Comment by anponnet** [comment]
> @anponnet You are removing an operation from a stable API... this is a breaking change. We will have to wait for ARM's feedback on this change.
> Why not bump up the API version here instead of modifying an existing Stable API version?

We are still under public preview and API/SDK are not published yet, GA is scheduled for March 27. We needed a stable version for that release, so releasing this APIs as stable. That is the reason we are not bumping up the version. The document was published a couple of weeks back.

**Comment by anponnet** [comment]
> @anponnet You are removing an operation from a stable API... this is a breaking change. We will have to wait for ARM's feedback on this change.
> Why not bump up the API version here instead of modifying an existing Stable API version?

We created swagger as stable, though service itself is preview because we are going GA on March 27. Having swagger as stable a few weeks near to GA will help us to have rest API documentation ready by GA

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Comment by praries880** [comment]
@RyanBensonMSFT Can you kindly take a look at this PR?

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2019-03-01/edgegateway.json
Comment (ravbhatnagar) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2019-03-01/edgegateway.json]: @anponnet - Has this API ever worked? It couldn't have since the URL is not well-formed as per ARM guideline. Was the PUT added by accident to swagger? Or do you actually have a PUT for this path? 
I checked the calls to this API and there is no traffic to this path in the last 45 days. 
Please confirm and then we can sign off. Even though theoretically its breaking change for the SDK, but my hunch is that it never would have worked. And so it may be ok.

**Review by ravbhatnagar — COMMENTED** [review]
@anponnet - please take a look at one comment

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2019-03-01/edgegateway.json
Comment (anponnet) [reply specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2019-03-01/edgegateway.json]: @ravbhatnagar , We had a PUT call on this path when we released our private preview which was not reviewed by ARM. Later on this API was removed and it is never in use. So there wont be any call to that API. Though API was removed, the Path was part of swagger and it went in our last swagger release. Our SDK (or even API documentation) is not yet released, so it is safe to remove this

Comment (ravbhatnagar) [reply specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2019-03-01/edgegateway.json]: Sounds good and thanks for confirming. @praries880 - this should be good to remove.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by praries880** [comment]
@ravbhatnagar  Thanks for signing off on this.

@anponnet  I have restarted the failed build for this : https://azuresdkci.westus2.cloudapp.azure.com/job/DotNetSwaggerToSdkCI/80/
I will merge this in once the build succeeds.

**Comment by praries880** [comment]
@anponnet  Closed and re-opemed the PR to kick off the CI process agian.

**Comment by praries880** [comment]
@sergey-shandar Can you look into the failure of the CI job?

**Comment by anponnet** [comment]
> @sergey-shandar Can you look into the failure of the CI job?

@sergey-shandar , Can you please help to get it merged Today. We are very near to GA and it is a blocker for anouncement.

**Review by praries880 — APPROVED** [review]

**Pull request merged**
